templates/zerver/help/delete-all-organization-data.md

```diff
@@ -0,0 +1,12 @@
+# Delete your organization's data
+
+Upon request by all owners of an organization hosted on Zulip Cloud,
+we can delete all organization data, including user accounts and information
+as well as messages. Please note that we have to preserve:
+
+* Old logs related to the organization
```

Comment:
Perhaps "Server logs necessary for investigating reports of abuse involving the organization."